tests/sentinel/run.tcl

```diff
@@ -22,6 +22,7 @@ proc main {} {
     spawn_instance redis $::redis_base_port $::instances_count {
         "enable-protected-configs yes"
         "enable-debug-command yes"
+        "save ''"
```

Comment:
don't we need to drop this one too?

Reply:
this is a redis instance, not a sentinel instance. do we need to drop it too?

Reply:
ohh sorry. i don't know the sentinel tests, and didn't bother to look at the code.
concluded that if it's in the sentinel folder then it should be reverted.